working/v3.0.0-draft2/authorization-code-flow/JARM-response-schema.json

```diff
@@ -0,0 +1,72 @@
+{
+    "title": "Version 1 of JARM response payload",
+    "$id": "http://apicentre.paymentsnz.co.nz/schemas/jarm-response-schema.json",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "required": [
+      "iss",
+      "code",
+      "aud",
+      "exp",
+      "state",
+      "ConsentId"
+    ],
```

Comment:
Optional for the string fields - adding `"minLength": 1` to make sure that e.g. code: "" doesn't validate.

Reply:
Added `"minLength": 1` to string fields